assets/js/vue/components/FundingCyclesList.vue

```diff
@@ -34,67 +45,81 @@
+                disableResGrpBtn: true,
+                disableProjDirBtn: true,
             }
         },
         methods: {
-            populateResearchGroups: function(event) {
-                this.researchGroups = [];
-                this.researchGroupDisabled = true;
+            populateResearchGroups: function(fundingCycleId) {
+                this.researchGroupOptions = [];
+                this.selectedResearchGroup = null;
                 this.fundingCycles.forEach(fundingCycle => {
-                    if (fundingCycle.id === Number(event.target.value)) {
-                        this.researchGroups = fundingCycle.researchGroups;
-                        this.researchGroupDisabled = false;
+                    if (fundingCycle.id === Number(fundingCycleId)) {
+                        fundingCycle.researchGroups.forEach(researchGroup => {
+                            this.researchGroupOptions.push({
+                                value: researchGroup.id,
+                                text: this.$options.filters.truncate(researchGroup.name, 100)
+                            })
+                        })
                     }
                 })
+                this.disableResearchGroups = this.researchGroupOptions.length <= 1;
             },
-            selectResearchGroup: function(event) {
-                if (event.target.value) {
-                    this.openResearchGroupLandingPage(event.target.value);
+            researchGroupButton: function() {
+                this.openResearchGroupLandingPage(this.selectedResearchGroup);
+            },
+            openResearchGroupLandingPage: function (researchGroupId) {
+                if (researchGroupId) {
+                    window.open("/research-group/about/" + researchGroupId, '_blank');
                 }
             },
-            populateProjectDirectors: function () {
+            populateFundingCycles: function () {
                 this.fundingCycles.forEach(fundingCycle => {
-                    fundingCycle.researchGroups.forEach(researchGroup => {
-                        researchGroup.projectDirectors.forEach(projectDirector => {
-                            if (this.projectDirectorIds.indexOf(projectDirector.id) === -1) {
-                                this.projectDirectorIds.push(projectDirector.id);
-                                this.makeProjectDirectorOption(researchGroup.id, projectDirector.name);
-                            } else {
-                                this.projectDirectorIds.push(projectDirector.id);
-                                this.makeProjectDirectorOption(
-                                    researchGroup.id,
-                                    projectDirector.name + ' - ' + researchGroup.shortName
-                                );
-                            }
-                        })
+                    this.fundingCycleOptions.push({
+                        value: fundingCycle.id,
```

Comment:
Here one could have used `text-field` and `value-field` in the corresponding b-form-select.

Reply:
I don't know whether it makes any difference, instead of using value and text, they have used custom names for them. we still have to push them to the options object